api/query/query.go

```diff
@@ -61,28 +61,30 @@ func (qh queryHandler) processInput(w http.ResponseWriter, r *http.Request) (*sh
 	return &filters, testRuns, summaries, nil
 }
 
-func (qh queryHandler) getVersionFiles(w http.ResponseWriter, r *http.Request) ([]string, error) {
-
+func (qh queryHandler) validateSummaryVersions(w http.ResponseWriter, r *http.Request) (bool, error) {
```

Comment:
ResponseWriter is not needed anymore in the function.
You could change the signature to pass in only `query string`

Reply:
True - I refactored it 👍